charts/kubestash-catalog/values.yaml

```diff
@@ -19,6 +19,9 @@ elasticsearch:
   restore:
     # Arguments to pass to `multielasticdump` command during restore process
     args: ""
+opensearch:
+  # If true, deploys Opensearch addon
+  enabled: true
```

Comment:
Add
```
  # optional argument to send multielasticdump
  backup:
    # Arguments to pass to `multielasticdump` command  during backup process
    args: ""
  restore:
    # Arguments to pass to `multielasticdump` command during restore process
    args: ""
```

Reply:
We are using the same functions for elasticsearch and opensearch. Shouldn't these values be taken from those functions (elasticsearch functions)?

Reply:
No, create separately